.prettierrc.cjs

```diff
@@ -1,4 +1,5 @@
 module.exports = {
   "singleQuote": true,
   "printWidth": 100,
+  "trailingComma": "es5"
```

Comment:
avoids big diff with prettier v3 new defaults

I would recommend a later PR to remove this custom config item and run eslint fix over everything to do commas the standard way

Reply:
Sounds good.